Utilities/OpenCorePackage/Package.command

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# script OpenCorePackage
+# OpenCorePackage by chris1111
+#
+
+# Use some vars
+PARENTDIR=$(dirname "$0")
+cd "$PARENTDIR"
+root_build="./OC-EFI"
+distrib_dir="./Distribution"
+resources_dir="./Resource"
+sript_uefi="./Script/UEFI"
+script_duet="./Script/Legacy"
+temp_dir="/Private/tmp/Package-DIR"
+root_dir="./OpenCore-Package"
+package_name="OpenCorePackage.pkg"
+duet_name="./OC-EFI/usr/standalone/i386/boot0"
+duetfile_name="Duet"
+
+# Delete build if exist
+find . -name '.DS_Store' -type f -delete
```

Comment:
Why is there an explicit handling of this file? It should be implicitly deleted as part of folder purge or just be left alone, no?